bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -2506,13 +2603,29 @@ bool BCStateTran::isMaxBlockIdInFetchRange(uint64_t blockId) const {
 
 bool BCStateTran::isRvbBlockId(uint64_t blockId) const {
   ConcordAssertGT(blockId, 0);
-  return isMaxBlockIdInFetchRange(blockId);
+  return ((blockId % config_.fetchRangeSize) == 0);
```

Comment:
Are we protected from the case of config_.fetchRangeSize being 0?

Reply:
I dont see an issue here.
I mean, nothign will work but we are not protected on any value of it if you mis-config the system. for example - we are not protected also for fetchRangeSize = 10^10
In terms of security, it's just a config parameter.
We do not have extensive parameter check.
We just apply when needed.